app/views/standard/addAdditionalMetadata.scala.html

```diff
@@ -1,18 +1,12 @@
 @* helper package contains inputRadioGroup therefore exclude from imports to prevent ambiguous import with view of the same name *@
+@import controllers.AddAdditionalMetadataController.File
+@import controllers.util.MetadataProperty.{description, end_date, foiExemptionCode}
+@import controllers.util._
 @import helper.{inputRadioGroup => _, _}
-@import java.util.UUID
+@import views.html.partials.{inputDate, inputText, _}
```

Comment:
Very minor thing but can you have this import as `@import views.html.partials._`

Reply:
There was some ambiguous imports due to the way we imported the helper package. So I've imported the helper package more explicitly, everything seems to still work.

Reply:
Cool, thanks. You've got some merge conflicts, I'll approve it once those are gone.